src/scripts.js

```diff
@@ -1,12 +1,40 @@
-// This is the JavaScript entry file - your code begins here
-// Do not delete or rename this file ********
-
-// An example of how you tell webpack to use a CSS (SCSS) file
 import './css/base.scss';
+import domUpdates from './domUpdates.js';
+
+import { fetchRooms,
+  fetchBookings,
+  fetchCustomers
+} from './apicalls';
 
-// An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
+import Customer from '../classes/Customer.js';
+import Hotel from '../classes/Hotel.js';
+
+// images
 import './images/overlook-carpet-top-down.png';
 
+// Global variables
+
+let customer;
+let hotel;
+
+
+const fetchAll = () => {
+  Promise.all([fetchRooms(), fetchBookings(), fetchCustomers()])
+  .then(data => {
+    customer = new Customer(data[2][0]);
+    hotel = new Hotel(data[0], data[1], data[2][0]);
+    getCustomerInformation()
+  })
+}
+
+const getCustomerInformation = () => {
+  hotel.getCurrentCustomerBookings();
+  hotel.calculateTotalCostOfAllCustomerBookings();
+  customer.getFirstName();
+  // console.log(customer.getFirstName())
+  domUpdates.displayTotalAmountSpentOnBookings(hotel.totalBookingCost);
+  domUpdates.displayCustomerInformation(hotel.currentCustomerBookings);
+  domUpdates.displayPersonalizedGreeting(customer.getFirstName(hotel.customer));
+}
 
-console.log('This is the JavaScript entry file - your code begins here.');
+window.addEventListener('load', fetchAll);
```

Comment:
Your promise looks correct, let me know if you run into any issues with API data flow -- happy to try and troubleshoot if I can.